### PR TITLE
updated readme with the cli tool details

### PR DESCRIPTION
**Pull request by amrrs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,35 @@ The fastest way to share your notebook with someone.
...
+
+There are two ways to share your Notebooks
+
+1. You can upload your notebook easily via the web interface at [notebooksharing.space](https://notebooksharing.space/) *(No Sign up required)*
+2. `nbss-upload` Command-Line Tool
+
+
+## CLI
+
+`nbss-upload` is available on [PyPI](https://pypi.org/project/nbss-upload/), and can be installed with pip.
```
Comment (yuvipanda) [inline README.md:22]: Link to GitHub as well, at github.com/notebook-sharing-space/nbss-upload?

Comment (amrrs) [reply README.md:22]: My bad I didn't see that. Should I just link it to the second point where command line is mentioned or Do you have any suggestions?

Comment (yuvipanda) [reply README.md:22]: Yeah, just linking it on the second point seems good :)

**Comment by amrrs** [comment]
Updated with the link and some formatting, please check @yuvipanda

**Comment by yuvipanda** [comment]
Thanks a lot, @amrrs! And congratulations on getting this PR merged!!! \o/

**Comment by amrrs** [comment]
Thank you that's my pleasure :)

**Pull request merged**